### DISCUSSION TITLE
Request to Add Email Address (@o.cnu.ac.kr) for Access to the Server

### DISCUSSION THREAD

**Discussion by ParkJongham** [RosettaCommons/rosetta#87] [ROSIE]
Hello,
I'd like to access ROSIE but I'm getting an error that I have to confirm my institution is academic. I currently study at Institute of Chungnam National University (https://plus.cnu.ac.kr/html/kr/). Thus, it would be great if both got marked as academic institutions.
Thank you and have a nice day,

Jongham Park

**Comment by lyskov** [comment]
Thank you for letting us know! This should be fixed now. Please feel free to re-open if you still can't login. Thanks,
